submissions/YanaGushlevskaya/html-css-popup/dist/index.html

```diff
@@ -0,0 +1,320 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>
+    
+  Pop-up menu
+
+  </title>
+  <link rel="stylesheet" href="css/main.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/png" href="img/kottans.png"/>
+</head>
+<body>
+  <header class="header">
+  <nav class="header-nav">
+    <ul class="menu reset-list">
+      <li class="menu__item">
+        <a href="#" class="menu__link reset-link">Gmail</a>
+      </li>
+      <li class="menu__item">
+        <a href="#" class="menu__link reset-link">Images</a>
+      </li>
+      <li class="menu__item">
+        <input type="checkbox" id="dropdown__input" class="dropdown__input">
+        <label for="dropdown__input" class="menu__link label">
+          <i class="fas fa-th"></i></label>
```

Comment:
Navigating through TAB does not focus the menu button.

Reply:
Could you please tell me how to fix this in a better way? As I understood the problem is in css-property:absolute.

Reply:
```CSS
.dropdown__input:focus + .label {
    //styles for focus state
}
```